Assets/Scripts/Model/GraphElementClass.cs

```diff
@@ -6,13 +6,16 @@
 namespace Assets.Scripts.Model
 {
   [Serializable]
-  public class NodeClass
+  public class GraphElementClass
   {
     [JsonProperty] public int Id { get; private set; }
     [JsonProperty] public string Name { get; private set; }
     [JsonProperty] public PrimitiveType? Shape { get; private set; }
     [JsonProperty] public string TexturePath { get; private set; }
     [JsonProperty] public float? Scale { get; private set; }
+    [JsonProperty] public LineEnding? LineEnding { get; private set; }
```

Comment:
I don't like that the node class has lineending which it never uses

Reply:
It's not node class. I have unified all the classes so they can all be defined in the json once and reused by all components. I have considered inheritance but that would add extra complexity.
Right now, if an element can't use a property - it doesn't and nothing bad happens.
I will ad an issue for this but again this requires design changes and consideration